app/Http/Controllers/AlbumController.php

```diff
@@ -753,7 +754,11 @@ public function getArchive(Request $request)
 						// Get extension of image
 						$extension = Helpers::getExtension($url, false);
 						// Set title for photo
-						$file = $title . $extension;
+						if ($prefix_url != 'raw/') {
```

Comment:
As it's the same block, I feel this should be the same test as L741, `$photo->type == 'raw'`. Or we could set a boolean up there e.g. `$raw = $photo->type == 'raw';`?

I'll try to get around to the rest, but I've just seen how long this diff is!